community/cypher/interpreted-runtime/src/main/scala/org/neo4j/cypher/internal/runtime/interpreted/commands/expressions/Add.scala

```diff
@@ -45,6 +45,9 @@ case class Add(a: Expression, b: Expression) extends Expression {
       case (x: IntegralValue, y: TextValue) => Values.stringValue(x.longValue() + y.stringValue())
       case (x: TextValue, y: FloatValue) => Values.stringValue(x.stringValue() + y.doubleValue())
       case (x: FloatValue, y: TextValue) => Values.stringValue(x.doubleValue() + y.stringValue())
+      case (x: TemporalValue[_,_], y: DurationValue) => x.plus(y)
+      case (x: DurationValue, y: TemporalValue[_,_]) => y.plus(x)
+      case (x: DurationValue, y: DurationValue) => x.add(y)
```

Comment:
I suggest to change `DurationValue.add` to `DurationValue.plus`, to adhere to the same pattern and the other values. Same with the other arithmetics on `DurationValue`.

Reply:
Since the signatures are slightly different `Temporal.plus(Duration) : Temporal` while `Duration.add(Duration) : Duration`, i think it does not hurt having them named differently.